afs/afs-core/src/main/java/com/powsybl/afs/DependencyCache.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public class DependencyCache<T> {
+public class DependencyCache<T extends ProjectNode> {
```

Comment:
Does it beak something in another project?

Reply:
No. Every dependency is always a ProjectNode